### PR TITLE
Update cache/origin docs to refer to the new authfile service and timer

### PR DESCRIPTION
**Pull request by matyasselmeci**
[SOFTWARE-5473](https://opensciencegrid.atlassian.net/browse/SOFTWARE-5473). I also mentioned that you could run an auth instance, unauth instance, or both, and also that you need different services on the origin for multiuser.

[SOFTWARE-5473]: https://opensciencegrid.atlassian.net/browse/SOFTWARE-5473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/data/stashcache/install-cache.md [hunk 1]
```diff
@@ -274,26 +294,28 @@ As a reminder, here are common service commands (all run as `root`):
 | Enable a service to start on boot       | `systemctl enable <SERVICE-NAME>`  |
 | Disable a service from starting on boot | `systemctl disable <SERVICE-NAME>` |
 
-### Public cache services
+### Public cache services (if running a public cache instance)
```
Comment (brianhlin) [inline docs/data/stashcache/install-cache.md:297]: Same comment here about the admonition + long heading name

**Review by brianhlin — COMMENTED** [review]
Comments made in the cache doc also apply to the origin doc since they appear to mostly be the same

File: docs/data/stashcache/install-cache.md [hunk 2]
```diff
@@ -253,7 +261,19 @@ The FQDN of the cache server that you registered in [Topology](#registering-the-
...
+    with the following contents:
+   
+        :::ini
+        [Service]
+        Environment=CACHE_FQDN=<Topology-registered FQDN>
+
+
+If you are running an authenticated cache instance:
+
+1.  Create the file `/etc/systemd/system/stash-authfile@stash-cache-auth.service.d/override.conf`
```
[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/stashcache/install-cache.md:276]: Yep if you leave off the stuff after the `@` (but keep the `@`) then it applies to all instances of the service, let's just have people do that.

**Review by brianhlin — APPROVED** [review]
LGTM! I think we should probably have a separate multiuser configuration section somewhere but I've probably scope-creeped this review enough :)

**Comment by brianhlin** [comment]
Link failures seem unrelated, merging

**Pull request merged**